### ISSUE TITLE
Ability to just clip the url title and description (without actually clipping page content for reader mode, or indexing it either)

### ISSUE THREAD

**Issue by ashutoshsaboo** [omnivore-app/omnivore#2175]
There are lots of authwalled (assume if someone is on intranet) and paywalled usecases where one wouldn't want Omnivore to simply read the internal confidential data. There should be a way to just add the url to omnivore (to use it as a cross platform bookmark manager), and let it just save basic metadata: like the url, title, description of the page to omnivore, but no more. It shouldn't clip the content either client side or server side, and also not index the content of the page to omnivore's search index for obvious confidentiality reasons (due to intranet). 

Here's my thought. Ideal UX would be to just have a small "confirm to clip and store page content" checkbox in the mobile app share popups as well as extension popups. This can be by default set to ticked as most people wouldn't have intranet usecases. If someone doesn't tick it, then app should respect to not clip the page content and also not index the page content. In the list for such URLs where checkbox wasn't ticked: just show the url, title, description of the page along with the manifest icon if it exists for the website that was clipped. And When such urls are clicked, just navigate to the actual web page rather than the reader mode (as there'd be nothing to show in the reader mode, given backend wouldn't have clipped anything). 

For the API similar can be done, to just accept an optional param whether to clip page content to omnivore or not, if that's passed as false explicitly - server shouldn't access the page. Otherwise if the param isn't passed or if it's set to true, the flow runs as-is as of today - making it backwards compatible too.

What do you think? Thoughts? Can this be please added? @jacksonh 

Thanks a bunch for your hard work btw with Omnivore! 🚀 It's the best read-it-later app out there, a hidden gem I'd say! :)

**Comment by jacksonh** [comment]
Yes this is an interesting concept, we have to think about how it can be done in an easy manner so it doesn't interfere with the regular workflow. I think we can use our SavePage API and just pass empty content.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
